src/main/java/com/twilio/http/TwilioRestClient.java

```diff
@@ -27,6 +28,7 @@ private TwilioRestClient(Builder b) {
         this.edge = b.edge;
         this.httpClient = b.httpClient;
         this.objectMapper = new ObjectMapper();
+        objectMapper.registerModule(new JavaTimeModule());
```

Comment:
Should add a comment as to why this is necessary. A test would be better if it's not too involved, but a comment would suffice for me.

Reply:
I'll add a comment for sure. If there are other tests which make assertions about what goes to stdout/stderr then I can look at fitting this into them but I'm not sure it's worth adding that kind of testing just for this. What do you think? (Testing which redirects stdout/err streams is a pain, in my experience, and I can't see another way to do it because there's nothing in-code to hook into for asserting on this)

Reply:
Agree it can be a pain. This looks good to me.